README.md

```diff
@@ -28,20 +29,21 @@ This relatively easy when there's already spare resources on the physical (Kuber
 takes careful coordination to move postgres instances from one node to another when the original
 node doesn't have the room.
 
-We've [tried a bunch](https://github.com/neondatabase/cloud/issues/1651) of existing tools and
-settled on the following:
+We've tried a bunch of existing tools and settled on the following:
 
 * Use [VM live migration](https://www.qemu.org/docs/master/devel/migration/index.html) to move running
   postgres instances between physical nodes
 * QEMU is used as our hypervisor
 * [NeonVM](https://github.com/neondatabase/autoscaling/tree/main/neonvm) orchestrates NeonVM VMs as custom resources in
```

Comment:
**minor suggestion (not blocking)**:  instead of wording "neonvm orchestrates neonvm vms as custom resources" we could use more common in k8s ecosystem wording like "neonvm is the controller for vm.neon.tech/v1 VirtualMachnine custom resource"